src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationCache.cs

```diff
@@ -178,6 +213,24 @@ public DateTimeOffset LastVerifiedAt
             }
         }
 
+        public DateTimeOffset InitializedAt
+        {
+            get
+            {
+                if (!initializedAtValue.HasValue)
+                {
+                    initializedAtValue = DateTimeOffset.Parse(InitializedAtString);
+                }
+
+                return initializedAtValue.Value;
+            }
+            set
+            {
+                InitializedAtString = value.ToString();
```

Comment:
All of these `DateTimeOffset.ToString()` calls need to specify the ISO 8016 format, eg. `2017-11-16T20:24:46+00:00`. As it is, they're outputting strings in whatever local configuration the system has, which may not be parseable by `Parse`.

Reply:
Originally fixed in #457 carried over here